procrustes/generalized.py

```diff
@@ -62,6 +62,11 @@ def generalized(array_list, ref=None, tol=1.e-7, n_iter=200, check_finite=True):
 
     This function implements the Equation (20) and the corresponding algorithm in  Gower's paper.
 
+    References
+    ----------
+    [1] J. C. Gower. Generalized Procrusted Analysis. Psychometrika, 40(1): 33-51, 1975.
+           doi: 10.1007/BF02291478.
+
```

Comment:
At some point, we decided not to include References in the docstrings, as they were already long.
Instead, we included all references in a table (on the website) summarizing all Procrustes methods. In this way, we could add a more comprehensive set of references. So, I will remove this.